packages/fxa-content-server/app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -300,6 +300,10 @@ const FxSyncChannelAuthenticationBroker = FxSyncAuthenticationBroker.extend(
           offeredEngines: loginData.offeredSyncEngines,
           declinedEngines: loginData.declinedSyncEngines,
         };
+      } else if (this.relier.get('service') === 'sync') {
+        // if service is already forced to be Sync, then for sign-in
+        // attempts we tell Desktop to just enable sync
+        loginData.services.sync = {};
```

Comment:
This will make sure desktop enables Sync after a sign in

Reply:
Tested this locally with Nightly and it doesn't seem to enable sync after reconnecting?

<img width="890" alt="Screen Shot 2019-10-09 at 8 32 02 PM" src="https://user-images.githubusercontent.com/1295288/66530034-fe7b4000-ead3-11e9-8abd-abcc89ac8787.png">

Reply:
Yeah requires a desktop patch, cc @mhammond